### DISCUSSION TITLE
How do i turn off this feature in the editor

### DISCUSSION THREAD

**Discussion by nstlgy** [zed-industries/zed#10158] [Feature Requests]
How do i stop zed from showing that `...data:`. I am not sure what its called or what is it for any help or info on this would be really appreciated.

<img width="587" alt="image" src="https://github.com/zed-industries/zed/assets/122057887/4d4a9534-1cf4-49d5-a73b-3ce112c37862">

**Comment by SomeoneToIgnore** [comment]
This is called "inlay hints" and could be turned off by adding 
```json
"inlay_hints": {
    "enabled": false
}
```
to your settings.

**Reply by nstlgy** [reply]
Thanks for the follow-up! Shortly after posting, I figured out the issue myself. i wish i couldve realised it before opening the discussion for it

**Comment by remorses** [comment]
I finally understand what inlay hints are now, an image like this should be added to the documentation
